labelbox/schema/model_run.py

```diff
@@ -565,6 +568,86 @@ def export_v2(
         task_id = res["taskId"]
         return Task.get_task(self.client, task_id)
 
+    def send_to_annotate_from_model(
+            self,
+            batch_name: str,
+            data_rows: DataRowIdentifiers,
+            destination_project: str,
+            task_queue_id: Optional[str],
+            exclude_data_rows_in_project: bool = False,
+            ontology_mapping: Dict[str, str] = None,
+            override_existing_annotations_rule:
+        ConflictResolutionStrategy = ConflictResolutionStrategy.KeepExisting,
+            priority: int = 5) -> Task:
+
+        mutation_str = """mutation SendToAnnotateFromMeaPyApi($input: SendToAnnotateFromMeaInput!) {
+                            sendToAnnotateFromMea(input: $input) {
+                              taskId
+                            }
+                          }
+        """
+
+        destination_task_queue = {
```

Comment:
Any way we can share some of the common input processing and query formatting code between the two methods?

Reply:
Possibly - we just need to find the right place to put these utility methods